doc/solving/solve_operations_network.rst

```diff
@@ -38,5 +38,5 @@ Rule ``solve_operations_network``
 
 |
 
-.. automodule:: solve_operations_network
-    :noindex:
+.. .. automodule:: solve_operations_network
```

Comment:
Is this double double dot needed?

Reply:
Same as the earlier situation. the extra double dots was used to comments the part out